docs/install/kubernetes.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: default
+title: Kubernetes
+parent: Install and Configure
+nav_order: 1
```

Comment:
This is should be either 5 (if you expect it to be standalone forever) or 10 (if you suspect the need for more pages on Kubernetes).

Reply:
Setting a separate page for Security settings, like Custom Users & Roles and Certificates, seems more ease on beginners, since with the current manifests users can already have a up and running installation.